### ISSUE TITLE
use git conventions for command names

### ISSUE THREAD

**Issue by billymoon** [akonwi/git-plus#17]
I am familiar with git, so when I want to stage files, I look for the git add command. In this plugin, I have to do git write - which does not mean anything to me.

If the command is `only` adding files to the staging index, the command could be called `add`, and perhaps have an alias `write/whatever`.

If the command is going to `add` and then also do some other operation, perhaps draft a commit message or something, in order to make sure user does not expect it is simply a `git add`, you could call it `stage`.

In a nutshell - I know and understand git terminology, and don't really want to learn a plugin specific set of names/concepts.
